fabric-resource-loader-v0/src/main/resources/assets/fabric-resource-loader-v0/lang/ko_kr.json

```diff
@@ -0,0 +1,4 @@
+{
+  "pack.source.fabricmod": "Fabric 모드",
+  "pack.source.builtinMod": "%s에 내장됨"
```

Comment:
Are you sure this works?
The %s maybe needs a space after.

Reply:
I don't think we need a space there.

Reply:
Is not displayed, however I think this is a vanilla problem not displaying long strings.

![Immagine 2022-10-16 190214](https://user-images.githubusercontent.com/32025549/196048401-397d6fe6-7a80-45ea-ad42-b5edc4531e81.png)
![Immagine 2022-10-16 185254](https://user-images.githubusercontent.com/32025549/196048185-935f1b67-4ca9-4558-99cd-a69ef0ba7d21.png)